### DISCUSSION TITLE
Include additional mounts dynamically

### DISCUSSION THREAD

**Discussion by alexanderilyin** [devcontainers/community#48] [General]
I'd like to be able to include additional mounts dynamically, for example "dot files" from home. In current `docker run …` approach I just use `find …` to discover files I want to bring to the container and that add additional `--volume …` dynamically. I wonder if I can make VSCode to do the same when using `devcontainer.json`.

**Comment by evilhamsterman** [comment]
I'd like the same thing, it's already in discussion apparently https://github.com/devcontainers/spec/issues/132
